BrainCloudClient/tests/TestTournament.cs

```diff
@@ -138,6 +138,26 @@ public void PostTournamentScore()
             LeaveTestTournament();
         }
 
+        [Test]
+        public void PostTournamentScoreUTC()
+        {
+            JoinTestTournament();
+
+            TestResult tr = new TestResult(_bc);
+
+            _bc.TournamentService.PostTournamentScoreUTC(
+                _leaderboardId,
+                _rand.Next(1000),
+                null,
+                (UInt64)((TimeZoneInfo.ConvertTimeToUtc(DateTime.UtcNow) -
```

Comment:
Same, I'm not C# expert. let's quickly bring @stevemrjones and make sure this is the legit way of doing it.

Reply:
Lets use the UTILs helper to help with this conversion.